### ISSUE TITLE
Disclaimer

### ISSUE THREAD

**Issue by egpbos** [NLeSC/rsd-instruction#62]
I think it may be good if we add a text somewhere that we welcome suggestions for adding missing data, or in other words, a disclaimer that the information is probably always incomplete. For instance, a former project partner might be browsing the site looking for their name (as people do) and find that it's missing, which will probably upset/disappoint them, which certainly won't improve our reputation and connections with the field in question. If we make it easy/obvious for them to send us a quick message about this (e.g. a "suggest correction" button/form), we can give them the credit they probably deserve and keep our network in good shape.

**Comment by jspaaks** [comment]
Good idea. I think there was talk about a 'feedback' button at some point, but I don't see it in the latest iteration of the designs. We do have a "questions or comments" link at the bottom of both the product page and the overview page, but it's not very prominent.
